launcher/ui/pages/global/AccountListPage.cpp

```diff
@@ -89,6 +89,9 @@ AccountListPage::AccountListPage(QWidget* parent) : QMainWindow(parent), ui(new
         ui->actionAddMicrosoft->setVisible(false);
         ui->actionAddMicrosoft->setToolTip(tr("No Microsoft Authentication client ID was set."));
     }
+
+    ui->checkBoxUseKeychain->setChecked(APPLICATION->settings()->get("UseKeychain").toBool());
+    connect(ui->checkBoxUseKeychain, &QCheckBox::toggled, this, [](bool value) { APPLICATION->settings()->set("UseKeychain", value); });
```

Comment:
Is there a reason why we should make this configurable?
Wouldn't it be better to store it on a keychain simply? That way, it will be harder for the user to change the value.

Reply:
I want to allow it to be disabled in case there are bugs such as https://github.com/frankosterfeld/qtkeychain/issues/240
(I'll probably make it enabled by default on macos and windows)

Reply:
Linux should be as a default as well. We'd just need to make sure we avoid the libsecret regression mentioned there in our packaging

There's also the plain text backend we could probably fall back on. Might simplify things on our end